utils/mariadb/test.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 
 # shellcheck disable=SC1090
-. "$CI_HELPER"
```

Comment:
This would seem to define the function `success()`, which `check_version()` is attempting to invoke:
```
/ci/utils/mariadb/test.sh: line 12: success: not found
```
So, it looks like one of the version checks actually succeeded.

A better way of debugging this might be to retain this line and add the following line:
```
set -x
```
Then we should see in more detail what is happening.

Reply:
If you want to omit the `$CI_HELPER`, then you should not only replace the `success` and `err_die` but also the `info`.